lib/components/narrative/default/default-itinerary.js

```diff
@@ -27,37 +32,42 @@ const LegIconWrapper = styled.div`
   height: 20px;
   padding-bottom: 6px;
   padding-left: 2px;
-  width: 20px;
+  width: 20px; 
```

Comment:
Why was the trailing space added?

Reply:
I ran prettier at some point which caused these changes. Removed in 513973b0fc2d4404796775573af65c8859c746a2